recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib
+
+./configure \
+    --enable-shared \
+    --enable-netcdf-4 \
+    --enable-dap \
+    --without-ssl \
+    --without-libidn \
+    --disable-ldap \
+    --prefix=$PREFIX
+
+make
+make check
+make install
+
+rm -rf $PREFIX/share
```

Comment:
Don't know the reason behind this. I believe this recipe is from the `anaconda-recipes` repostory.